profile.md

```diff
@@ -486,10 +486,14 @@ Examples values of the `scope` claim:
 
 ### Group Based Authorization: wlcg.groups 
 
-Authorization may be based on the `wlcg.groups` claim. The value of the `wlcg.groups` claim is an ordered JSON array of case-sensitive strings reflecting the VO groups of which the token subject is a member. 
 
-`wlcg.groups `semantics are equivalent to existing VOMS groups. VOMS roles should be considered as optional (i.e. returned only if requested) `wlcg.groups`; selection of optional groups is discussed below.
+Authorization may be based on the `wlcg.groups` claim. The value of the `wlcg.groups` claim is an ordered JSON array of case-sensitive strings reflecting the assertion, by the issuer, of group membership of the subject of the token.
 
+Relying parties MUST only consider the group membership asserted in the token for authorization decisions.  Particularly, membership in a child group (`/cms/uscms`) does not implicitly imply membership in the parent group (`/cms`) unless the parent group is explicitly listed in the token.
+
+The token does not need to list all groups the user has access to; relying parties MUST utilize only the asserted groups in the implied order.  The `wlcg.groups` claim is optional and may either be empty or not provided at all.  The user may provide input on the contents and ordering of this claim; this is covered [below](#scope-based-attribute-selection).
```

Comment:
> The token does not need to list all groups the user has access to

I may have missed past discussions, but what is the rationale? What problem are you trying to solve wrt to the status quo?

In any case, the section "Scope-based Group Selection" would also need to be rewritten.

Reply:
The rationale would be that tokens can/should be made as lean as possible to:

1. mitigate potential concerns about their sizes (there could be a large number of groups);
2. do not bring contents that are totally irrelevant to the authZ decision (but have to be parsed, forwarded and logged nonetheless).

Reply:
It sounds as premature optimization to me, but at least it's clear now. This requires changes in the token issuer, that's why I asked to rewrite also the following section.